gst/nnstreamer/tensor_converter/README.md

```diff
@@ -26,13 +33,14 @@ From higher priority
 
 ## Sink Pads
 
-One "Always" sink pad exists. The capability of sink pad is ```video/x-raw```, ```audio/x-raw```, ```text/x-raw``` and ```application/octet-stream```.
+One "Always" sink pad exists. The capability of sink pad is ```video/x-raw```, ```audio/x-raw```, ```text/x-raw```, ```application/octet-stream```, and ```other/tensors-flexible```.
 
 ## Source Pads
 
-One "Always" source pad exists. The capability of source pad is ```other/tensor```. It does not support ```other/tensors``` because each frame (or a set of frames consisting a buffer) is supposed to be represented by a **single** tensor instance.
+One "Always" source pad exists. The capability of source pad is ```other/tensor```, ```other/tensors```, and ```other/tensors-flexible```.
 
-For each outgoing frame (on the source pad), there always is a **single** instance of ```other/tensor```. Not less and not more.
+Note that, only octet-stream supports configuring multiple tensors in outgoing buffer.
```

Comment:
Serialized buffers(flatbuf, flexbuf and protobuf) and custom converter also support multiple tensors.

Reply:
update description about default caps and sub-plugin
- only octet in default caps supports ... 
- subplugin and custom converter